dsmr_notification/services.py

```diff
@@ -86,6 +86,14 @@ def send_notification(message, title):
                 'description': message
             }
         },
+        NotificationSetting.NOTIFICATION_TELEGRAM: {
+            'url': NotificationSetting.TELEGRAM_API_URL + notification_settings.telegram_api_key + '/sendMessage',
```

Comment:
Deze string concats mag je wel even aanpassen zodat ze `string.format()` gebruiken. Dat is net iets netter en robuuster dan `+`'s.

```suggestion
            'url': '{}{}/sendMessage'.format(NotificationSetting.TELEGRAM_API_URL, notification_settings.telegram_api_key),
```